From 51abdfcd2ece613a497945ab39144267ec4bb33f Mon Sep 17 00:00:00 2001
From: Ben Peled <bpeled@marvell.com>
Date: Tue, 30 Apr 2019 17:28:22 +0300
Subject: [PATCH 198/386] arm64: dts: marvell: change ap807 syscontrol clock to
 ap807-clock

Change-Id: I872c877fd15b407d28eb12e15b905415467aac6d
Signed-off-by: Ben Peled <bpeled@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/8306
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-ap806.dtsi | 7 +++++++
 arch/arm64/boot/dts/marvell/armada-ap807.dtsi | 7 +++++++
 arch/arm64/boot/dts/marvell/armada-ap80x.dtsi | 5 -----
 3 files changed, 14 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
index 175c64a73484..b69f478c2c90 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
@@ -24,6 +24,13 @@
 					#clock-cells = <1>;
 				};
 			};
+
+			ap_syscon0: system-controller@6f4000 {
+				ap_clk: clock {
+					compatible = "marvell,ap806-clock";
+					#clock-cells = <1>;
+				};
+			};
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-ap807.dtsi b/arch/arm64/boot/dts/marvell/armada-ap807.dtsi
index 9a919fa162ee..f6aac88f0d5d 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap807.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap807.dtsi
@@ -63,6 +63,13 @@
 					#clock-cells = <1>;
 				};
 			};
+
+			ap_syscon0: system-controller@6f4000 {
+				ap_clk: clock {
+					compatible = "marvell,ap807-clock";
+					#clock-cells = <1>;
+				};
+			};
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi b/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi
index 027317a3dbac..33a9e8b33b10 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi
@@ -404,11 +404,6 @@
 				compatible = "syscon", "simple-mfd";
 				reg = <0x6f4000 0x2000>;
 
-				ap_clk: clock {
-					compatible = "marvell,ap806-clock";
-					#clock-cells = <1>;
-				};
-
 				ap_pinctrl: pinctrl {
 					compatible = "marvell,ap806-pinctrl";
 
-- 
2.17.1

